docs/source/host_challenge.md

```diff
@@ -125,37 +131,62 @@ Finally run the `./run.sh` script in the bundle. It will generate a `challenge_c
 
-Steps to create a Code upload based challenge is very similar to what it takes to create a [prediction upload based challenge](host_challenge.html#host-a-prediction-upload-based-challenge).
+Steps to create a code-upload based challenge are somewhat similar to what it takes to create a [prediction upload based challenge](host_challenge.html#host-a-prediction-upload-based-challenge).
 
-We have created a sample challenge configuration that we recommend you to use to get started. Use [EvalAI-Starters](https://github.com/Cloud-CV/EvalAI-Starters) template to start. See [this](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) on how to use a repository as template.
+We have created a sample challenge repository that we recommend you to use to get started. Use [EvalAI-Starters](https://github.com/Cloud-CV/EvalAI-Starters) template to start. See [this](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) on how to use a repository as template.
 
 ### Step 2: Edit challenge configuration
 
 Open [`challenge_config.yml`](https://github.com/Cloud-CV/EvalAI-Starters/blob/master/challenge_config.yaml) from the repository that you cloned in step-1. This file defines all the different settings of your challenge such as start date, end date, number of phases, and submission limits etc. Edit this file based on your requirement.
 
-Make sure that following fields are set correctly:
+Please ensure the following fields are set to the following values for code-upload based challenges:
 
-- `remote_evaluation` is set to `True`
-- `is_docker_based` is set to `True`
+- `remote_evaluation : True`
+- `is_docker_based : True`
 
-For reference to the fields, refer to the [challenge configuration reference section](configuration.html).
+In order to perform evaluation, you might also need to create an EKS cluster on AWS. This is because we expect to use docker containers for both - the agent, and the environment. See [AWS Elastic Kubernetes Service](https://aws.amazon.com/eks/) to learn more about what EKS is and how it works.
 
-### Step 3: Edit evaluation script
+We need the following details for the EKS cluster in order to perform evaluations in case you are using your own AWS account:
 
-Next step is to edit the challenge evaluation script that decides what metrics the submissions are going to be evaluated on for different phases.
+- `aws_account_id: <AWS Account ID>`
+- `aws_access_key_id: <AWS Access Key ID>`
+- `aws_secret_access_key: <AWS Secret Access Key>`
+- `aws_region: <AWS Region>`
 
-Please refer to the [writing evaluation script](evaluation_scripts.html) to complete this step.
+These details need to be added to the challenge configuration.
+
+For reference to the other fields, refer to the [challenge configuration reference section](configuration.html).
```

Comment:
These are not part of the serializer so they can't add it to the challenge config. Let's remove this